### PR TITLE
Updated ESA Scale Document

### PR DESCRIPTION
**Pull request by marvinbuss**
**This PR fixes**
Updated ESA Scalability Document with comments from here: https://github.com/Azure/data-management-zone/pull/204

### CODE REVIEW

**Comment by marvinbuss** [comment]
@esbran & @amanjeetsingh please submit more enhancements directly to this branch

File: docs/guidance/EnterpriseScaleAnalytics-SelfService.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The following factors should be considered when deciding about how many Data Lan
 
 Considering all these factors, an enterprise should target no more than 15 Data Landing Zones as there is also some management overhead attached to each Data Landing Zone.
 
-It is important to emphasize that Data Landing Zones are not creating Data Silos within an organization, as the recommended network setup in ESA enables secure and in-place data sharing across Landing Zones and therefore enables innovation across data domains and business units. Please read the [network design guidance](/docs/guidance/EnterpriseScaleAnalytics-NetworkArchitecture.md) to find out more about how this is achieved.
```
Comment (amanjeetsingh) [inline docs/guidance/EnterpriseScaleAnalytics-SelfService.md:47]: "Data Silos" change to "data silos". (lowercase).

File: docs/guidance/EnterpriseScaleAnalytics-SelfService.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ The following factors should be considered when deciding about how many Data Lan
...
+It is important to emphasize that Data Landing Zones are not creating Data Silos within an organization, as the recommended network setup in ESA enables secure and in-place data sharing across Landing Zones and therefore enables innovation across data domains and business units. Please read the [network design guidance](/docs/guidance/EnterpriseScaleAnalytics-NetworkArchitecture.md) to find out more about how this is achieved. The same is true for the identity layer. Since a single Azure AD tenant is recommended to be used, identities can be granted access to data assets within multiple Data Landing Zones. For the actual authorization process of users and identities to data assets it is recommended to leverage Azure Active Directory Entitlement Management and Access Packages.
```
Comment (amanjeetsingh) [inline docs/guidance/EnterpriseScaleAnalytics-SelfService.md:47]: "Data Silos" change to "data silos".

**Review by amanjeetsingh — COMMENTED** [review]
minor update otherwise good to go

File: docs/guidance/EnterpriseScaleAnalytics-SelfService.md (hunk 2 above)
Comment (marvinbuss) [reply docs/guidance/EnterpriseScaleAnalytics-SelfService.md]: DONE

**Review by sasever — APPROVED** [review]
LGTMT

**Review by marvinbuss — COMMENTED** [review]
LGTM!

**Pull request merged**